package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "qr-online",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
What about this?
```suggestion
  "version": "1.1.0",
```

Reply:
There aren't so many changes, so I think I prefer 1.0.1.